package.json

```diff
@@ -8,60 +8,45 @@
     "src"
   ],
   "engines": {
-    "node": ">=10"
+    "node": ">=18"
   },
   "scripts": {
-    "start": "tsdx watch",
-    "build": "tsdx build --format cjs",
-    "test": "tsdx test",
-    "lint": "tsdx lint",
-    "prepare": "tsdx build",
-    "size": "size-limit",
-    "analyze": "size-limit --why"
-  },
-  "peerDependencies": {},
-  "husky": {
-    "hooks": {
-      "pre-commit": "tsdx lint"
-    }
-  },
-  "prettier": {
-    "printWidth": 80,
-    "semi": true,
-    "singleQuote": true,
-    "trailingComma": "es5"
+    "lint": "eslint .",
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest --coverage"
   },
   "name": "hathor-wallet-service-sync_daemon",
   "author": "André Abadesso",
-  "module": "dist/hathor-wallet-service-sync_daemon.esm.js",
-  "size-limit": [
-    {
-      "path": "dist/hathor-wallet-service-sync_daemon.cjs.production.min.js",
-      "limit": "10 KB"
-    },
-    {
-      "path": "dist/hathor-wallet-service-sync_daemon.esm.js",
-      "limit": "10 KB"
-    }
-  ],
+  "module": "dist/index.js",
   "devDependencies": {
-    "@size-limit/preset-small-lib": "^4.10.2",
     "@types/lodash": "^4.14.172",
-    "@types/node": "^17.0.21",
-    "husky": "^6.0.0",
-    "size-limit": "^4.10.2",
-    "tsdx": "^0.14.1",
+    "@types/mysql": "^2.15.21",
+    "@types/node": "^17.0.45",
+    "@typescript-eslint/eslint-plugin": "^6.7.3",
+    "@typescript-eslint/parser": "^6.7.3",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-jest": "^27.4.0",
+    "sequelize-cli": "^6.6.1",
     "tslib": "^2.1.0",
-    "typescript": "^4.2.3"
+    "typescript": "^4.9.5"
   },
   "dependencies": {
-    "@hathor/wallet-lib": "^0.20.3",
-    "aws-sdk": "^2.878.0",
-    "axios": "^0.21.1",
+    "@hathor/wallet-lib": "^1.0.2",
+    "@types/jest": "^29.5.4",
+    "@types/ws": "^8.5.5",
+    "assert": "^2.1.0",
+    "aws-sdk": "^2.1454.0",
+    "axios": "^0.21.4",
     "dotenv": "^8.2.0",
+    "jest": "^29.6.4",
```

Comment:
Shouldn't jest be a dev dependency?

Reply:
Same for the @types/jest

Reply:
Fixed in aca440380aea3b7eb464f02a9d2a3a10fd1b9eb2